connector/connect/server/src/main/scala/org/apache/spark/sql/connect/dsl/package.scala

```diff
@@ -513,6 +513,25 @@ package object dsl {
         freqItems(cols.toArray, support)
 
       def freqItems(cols: Seq[String]): Relation = freqItems(cols, 0.01)
+
+      def sampleBy(col: String, fractions: Map[Any, Double], seed: Long): Relation = {
```

Comment:
If this file is for testing, should it be moved to the `test` directory?

Reply:
This file was used to be here because we assumed that the DSL here can be used to build Scala client. But now this DSL is purely used for testing, then probably we can move it.

Reply:
I am fine to move `dsl` to `test` if nothing is broken